packages/botonic-plugin-hubtype-analytics/src/types.ts

```diff
@@ -0,0 +1,123 @@
+export enum EventName {
+  botAgentRating = 'bot_agent_rating',
+  botChannelRating = 'bot_channel_rating',
+  botFaqUseful = 'bot_faq_useful',
+  botRating = 'bot_rating',
+  botFaq = 'bot_faq',
+  botStart = 'bot_start',
+  botOpen = 'bot_open',
+  botAiModel = 'bot_ai_model',
+  botKeywordsModel = 'bot_keywords_model',
+  fallback = 'fallback',
+  handoffOption = 'handoff_option',
+  handoffSuccess = 'handoff_success',
+  handoffFail = 'handoff_fail',
```

Comment:
do we have an agreement to do this in snake_case or kebab-case? I think we are messing things through all the codebase, not blaming you!!